bindings/java/src/org/sleuthkit/datamodel/HostAddressManager.java

```diff
@@ -774,4 +782,44 @@ private static boolean isIPv6(String ipAddress) {
 
 		return false;
 	}
+	
+	/**
+	 * Normalizes an address.
+	 *
+	 * @param address
+	 *
+	 * @return Normalized address.
+	 */
+	private static String getNormalizedAddress(String address) {
```

Comment:
@raman-bt  shouldn't this also do the toLower() since it returns a normalized form?

Reply:
It intentionally does NOT convert to lowercase so that we can preserve the case and leave it to caller to convert if necessary.  There's a comment to that effect on getNormalizedIPV6Address() just a few lines below.  

I will add the comment here as well.

Reply:
Ok. that makes sense. I did not notice the other comment. sorry.